scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -75,15 +78,16 @@ workflow GvsCalculatePrecisionAndSensitivity {
         output_basename = output_sample_basename
     }
 
-    call Add_AS_MAX_VQSLOD_ToVcf {
+    call Add_AS_MAX_VQS_Score_ToVcf {
```

Comment:
is it VQS score? not VQSR? Doesn't the S for "score" ?

Reply:
Yeah - This label is very specific to this WDL and I was just looking for a generic term to use for lod and calibration_sensitivity. I'll change it up.